faf-java-server-app/src/main/java/com/faforever/server/game/GameService.java

```diff
@@ -836,38 +895,29 @@ protected void onGameEnded(Game game) {
   }
 
   /**
-   * Returns the active game with the specified ID, if such a game exists. A game is considered active as soon as it's
-   * being hosted and until it finished.
+   * Takes a map of armyId->ArmyResult and maps all army ids to player ids.
```

Comment:
Javadoc is HTML, so `->` might fail to render properly.

You can use `{@code armyId -> ArmyResult}`.

And spelling is `IDs`